pgrx/src/atomics.rs

```diff
@@ -7,30 +7,32 @@
 //LICENSE All rights reserved.
 //LICENSE
 //LICENSE Use of this source code is governed by the MIT license that can be found in the LICENSE file.
-use once_cell::sync::OnceCell;
+#![deny(unsafe_op_in_unsafe_fn)]
+use std::cell::UnsafeCell;
 
 pub struct PgAtomic<T> {
-    inner: OnceCell<*mut T>,
+    inner: UnsafeCell<*mut T>,
 }
 
 impl<T> PgAtomic<T> {
     pub const fn new() -> Self {
-        Self { inner: OnceCell::new() }
+        Self { inner: UnsafeCell::new(std::ptr::null_mut()) }
     }
 }
 
 impl<T> PgAtomic<T>
 where
     T: atomic_traits::Atomic + Default,
 {
-    pub fn attach(&self, value: *mut T) {
-        self.inner.set(value).expect("This PgAtomic is not empty, can't re-attach");
+    /// SAFETY: Can only be called from Postgres its shared memory init hook
```

Comment:
...was slow to notice this typo. The extra "its"
```suggestion
    /// SAFETY: Must only be called from inside the Postgres shared memory init hook
```

Reply:
Applied everywhere